docs/DEBUG_COMMANDS.md

```diff
@@ -27,6 +27,13 @@ For more information, please see the source code at [`Framework::ParseSearchQuer
 - `?isolines`: Enable the isolines layer.
 - `?no-isolines`: Disable the isolines layer.
 
+### 3D mode (for the Qt desktop app only)
+- `?3d`: Enable 3D (perspective) mode.
+- `?b3d`: Enable 3D buildings.
+- `?2d`: Disable 3D mode and buildings.
```

Comment:
Would it be easier to add these settings to QT app preferences?

Reply:
For sure!
Otherwise it took me awhile to find out how to switch to 2D mode..

Reply:
Let's create an issue )